subsys/bluetooth/audio/vocs.c

```diff
@@ -63,10 +64,16 @@ static ssize_t write_location(struct bt_conn *conn, const struct bt_gatt_attr *a
 		return BT_GATT_ERR(BT_ATT_ERR_INVALID_ATTRIBUTE_LEN);
 	}
 
-	memcpy(&inst->srv.location, buf, len);
-	BT_DBG("%02x", inst->srv.location);
+	new_location = sys_get_le32(buf);
+	if (new_location == BT_AUDIO_LOCATION_NOT_ALLOWED ||
+	    new_location >= BT_AUDIO_LOCATION_RFU_START) {
+		BT_WARN("Unsupported location");
+		return 0;
+	}
```

Comment:
Do you know if there's any requirement that the VOCS location shall be a subset of the locations in PACS/capabilities.c?

Reply:
I couldn't find anything about such a requirement in spec - but this would seem logical for the VOCS to be a subset of what we can actually support.

Reply:
Can't find anything w.r.t. this in CAP or TMAP. Might have been an oversight from the GAWG.

Reply:
Nevertheless, let's think of a situation when we have ASE Left Front but VOCS is Right Back. Who are we to prevent the user from creating his configuration - however he wants to set it up, should we prohibit this and limit this freedom - I wouldn't like that personally. What do you think?